examples/sampling/graphbolt/lightning/node_classification.py

```diff
@@ -211,7 +211,7 @@ def val_dataloader(self):
     dataset = gb.BuiltinDataset("ogbn-products").load()
     datamodule = DataModule(
         dataset,
-        [15, 10, 5],
+        [10, 10, 10],
```

Comment:
Please kindly relook my description. Since, the changes make the code consistent with DGL example.

Reply:
I mean, why not change those [10, 10, 10] to [15, 10, 5]?

Reply:
Understood.

However, I believe:
- We should rely on tests to verify the correct order of `fanouts` instead of utilizing checks that lack enforceability.
- Altering the `fanouts` in the original code might seem straightforward, but it could lead us to address numerous potential consistency issues, as various documents, results, and benchmarks might be contingent on the existing experimental setup.

I consider both approaches to be commendable. Yet, I'm inclined to think that implementing the requisite unit tests would be a more reasonable solution.